### DISCUSSION TITLE
Join our growing community on Slack!

### DISCUSSION THREAD

**Discussion by pycaret** [pycaret/pycaret#1003] [General]
# Join the growing PyCaret community on Slack!

Come say hi, share your ideas, or meet the team that is building the next generation of open-source machine learning and data science product.

EDIT - Link added (04/04/2021) - https://join.slack.com/t/pycaret/shared_invite/zt-nm02k73a-PSDo5lwmQ6evlFCPRxsKKA

**Comment by garyee** [comment]
It says: This Link is no longer active...

**Reply by pycaret** [reply]
Sorry. Here is the new link: https://join.slack.com/t/pycaret/shared_invite/zt-nm02k73a-PSDo5lwmQ6evlFCPRxsKKA

**Reply by ngupta23** [reply]
Here is the [latest updated invitation link](https://join.slack.com/t/pycaret/shared_invite/zt-qc4pfcw5-OLV3UFF~DzMHFyaJeFKw~A)

**Reply by AlxZed** [reply]
Expired as well, could you please update?

**Comment by pycaret** [comment]
I have now created a permanent link to the Slack channel. 

Join our Slack: https://join.slack.com/t/pycaret/shared_invite/zt-row9phbm-BoJdEVPYnGf7_NxNBP307w

**Comment by ellacharmed** [comment]
Hiya. Is there a valid Slack link? All the sites have the same "no longer active" for Slack.
